tasklist/app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<a class="create_form">
```

Comment:
This seems like a very odd thing to have in an anchor tag. Usually we just use text or an image, not sure what happens when you use a `form` inside.